content/docs/setup/kubernetes/download/index.md

```diff
@@ -51,3 +51,50 @@ To use the Istio release Helm chart repository, add the Istio release repository
 {{< text bash >}}
 $ helm repo add istio.io https://storage.googleapis.com/istio-release/releases/{{< istio_full_version >}}/charts/
 {{< /text >}}
+
+## istioctl Auto-Completion
+
+If you are using Bash or Zsh, `istioctl` includes a tab completion file that you can use. 
+
+### istioctl on Linux
+
+Both the Bash and Zsh completion files are located in the `tools` directory. Copy the preferred completion file to your home directory, and add the following line to source the `istioctl` completion file in your `.bashrc` file. For example with Bash, you will be adding the following line:
+
+```
```

Comment:
Thoughtout, please replace backticks with {{< text bash >}} and add a $ to command-lines.